backend/tests/TeamEventsAPI.test.ts

```diff
@@ -0,0 +1,88 @@
+import { PermissionError } from '../src/errors';
+import { teamEventsCollection } from '../src/firebase';
+import {
+  createTeamEvent,
+  deleteTeamEvent,
+  getAllTeamEvents,
+  updateTeamEvent
+} from '../src/team-eventsAPI';
+import jaggerData from './data/jagger-profile.json';
+
+const adminUser = { email: 'hl738@cornell.edu' } as IdolMember;
+const nonAdminUser = { email: 'aa2235@cornell.edu' } as IdolMember;
+
+const testTeamEvent = {
+  name: 'test',
+  date: '11/26/2020',
+  numCredits: '1',
+  hasHours: false,
+  membersPending: [jaggerData],
+  membersApproved: [],
+  uuid: 'test123'
+};
+
+afterAll(async () => deleteTeamEvent(testTeamEvent, adminUser));
+
+test('created team event with permission', async () => {
+  await createTeamEvent(testTeamEvent, adminUser).then((teamEvent) => {
+    expect(teamEvent.name).toEqual('test');
+  });
+});
+
+test('does not create team event because user does not have permission', async () => {
+  try {
+    await createTeamEvent(testTeamEvent, nonAdminUser);
+    throw new Error();
+  } catch (error) {
+    expect(error).toEqual(new PermissionError('does not have permissions to create team event'));
+  }
+});
```

Comment:
I think you can use the `toThrow()` function from jest to test for expected errors
ex.
```
expect(Promise.reject(createTeamEvent(testTeamEvent, nonAdminUser))).rejects.toThrow(
    Errors.PermissionError
);
```